# Parametri generali
min_particles: 500
max_particles: 2000
update_min_d: 0.2
update_min_a: 0.5
initial_pose_x: 0.0   # Mettilo uguale alla posa iniziale del robot1 in sim_config.yaml
initial_pose_y: 0.5   # Mettilo uguale alla posa iniziale del robot1 in sim_config.yaml
initial_pose_a: 0.0   # Mettilo uguale alla posa iniziale del robot1 in sim_config.yaml

# Frame IDs - DEVONO corrispondere a quelli pubblicati dal tuo simulatore
odom_frame_id: "robot1/odom"
base_frame_id: "robot1/base_link"
global_frame_id: "map"

# Modello del sensore laser
laser_model_type: "likelihood_field"
laser_z_hit: 0.95
laser_z_rand: 0.05
laser_sigma_hit: 0.2
laser_max_beams: 100